### PR TITLE
Add dotenv support to `dispatch run` and `dispatch login` commands

### PR DESCRIPTION
**Pull request by chicoxyzzy**
This PR adds a `--env-file` option support similar to [recently introduced Node.js's `--env-file`](https://nodejs.org/en/learn/command-line/how-to-read-environment-variables-from-nodejs).
Later, when we can add this file to `dispatch init` under some flags to autogenerate this file and avoid explicit export of environment variables for local development. It's also could be handy for adding rules like `make run-local` which run project against local development environment and other situation.

⚠️ Variables from the file do not override the ones passed with command line (default `github.com/joho/godotenv` behavior). I think this correct behavior, but it worth disacussing
⚠️ ~~I've realized to only override env variables on `PreRunE` Cobra hook to avoid FS operations on dispatch commands other than `run` and `login` to make them work faster, but I can change that to always load config as @achille-roussel proposed. The `--env-file` is a global option at the moment. We may want to make it non-global or leave it global and always load config. cc  @achille-roussel~~ The option is global and it always loads a file when provided

### CODE REVIEW

**Comment by chicoxyzzy** [comment]
I think that it would be super useful to add test coverage for the feature

File: cli/env_file.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	"github.com/joho/godotenv"
+)
+
+func loadEnvFromFile(path string) error {
+	if path != "" {
+		absolutePath, err := filepath.Abs(path)
+		if err != nil {
+			return fmt.Errorf("failed to get absolute path for %s: %v", path, err)
+		}
+		if godotenv.Load(path) != nil {
```
Comment (achille-roussel) [inline cli/env_file.go:17]: Let's not discard the error here and inject it in the log message below?

**Review by achille-roussel — COMMENTED** [review]
If I'm understanding correctly, the priority order will be:
- command line
- local env vars
- .env file

Is that right?


I would prefer to either make the option local to the commands it applies to, or always load the file if it's a global option. I like the "always available" if possible, it keeps the mental model simple, but also fine if we pick the other approach as a first step.

**Comment by chicoxyzzy** [comment]
> If I'm understanding correctly, the priority order will be:
> 
> * command line
> * local env vars
> * .env file
> 
> Is that right?

Yes, that's correct

**Comment by chicoxyzzy** [comment]
> I would prefer to either make the option local to the commands it applies to, or always load the file if it's a global option. I like the "always available" if possible, it keeps the mental model simple, but also fine if we pick the other approach as a first step.

The option is always global now

**Review by achille-roussel — APPROVED** [review]
Let's move env_file.go into config.go and get this merged!

Nice work 👏

**Comment by chicoxyzzy** [comment]
I've just made last changes and tested. It's ready to merge now

**Pull request merged**